ha4/templates/posts.html

```diff
@@ -0,0 +1,7 @@
+<html>
+    <form action='/post/new_post' method = "post">
+        author<input name = "name" type = "text" class="text">
+        title<input name = "Post_title" type = "text" class="text">
+        <p>content<input name = "content" type="text" style="text;width:500px;height:400px;" ></p>
+    </form> 
```

Comment:
to process submition you need something like this:
\< input type="submit" value="Submit"\>